server/pbenchinacan/container-build.sh

```diff
@@ -1,24 +1,29 @@
-#!/usr/bin/env bash
+#!/usr/bin/env bash -x
```

Comment:
You're not keeping the `-x`, are you?

Reply:
I was going to...if we don't, then we need to add error handling elsewhere....

FWIW, this is actually not a change in behavior -- see old line 7 -- this is just a change in syntax, as it were.

Reply:
Thanks to [Peter's dope-slap](https://github.com/distributed-system-analysis/pbench/pull/3214#discussion_r1091139223), I see that that should have been `-e`, not `-x`....  😊